kani-compiler/src/kani_middle/stubbing/mod.rs

```diff
@@ -51,7 +51,7 @@ pub fn validate_instance(tcx: TyCtxt, instance: Instance) -> bool {
         checker.visit_body(&item.body());
         checker.is_valid()
     } else {
-        true
+        instance.body().is_some()
```

Comment:
I agree that we shouldn't need this, but in the one place that invokes `validate_instance` we do `instance.body().unwrap()` afterwards. That now throws an exception that `instance.body()` is `None`. But it's likely fixing a symptom rather than the root cause.

Reply:
I am seeing different crashes related to evaluating the static initializer. I wonder what has changed.

Reply:
The implementation of `Instance` itself has changed?

Reply:
https://github.com/rust-lang/rust/pull/118681 fixes the root cause here. I'll create a workaround and try to update your branch

Reply:
Should we just wait for that to land and then skip ahead to the ensuing toolchain date?